engine/src/application.cpp

```diff
@@ -7,9 +7,11 @@
 namespace gen {
 
 	void Application::run() {
-        while (!m_window.shouldClose()) {
```

Comment:
Why has this been removed?

Reply:
all the changes to application have been made so I could have the main loop inside of the main function

Reply:
I don’t don’t see the main loop inside of the main function? From what I can tell currently there is no main loop anymore?

Reply:
oh my yeah I removed it to make the merge conflict easier my bad I'm just going to change back application